### PR TITLE
23 feature post review chekurkov

### PR DESCRIPTION
**Pull request by Dmitry528**
Please check this pull request like in Description in SLACK!!!

### CODE REVIEW

File: controllers/reviews-controller.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      res.status(200).send(data);
+    })
+    .catch(() => {
+      res.status(404).send({ message: '404 Not found' });
+    });
+};
+
+exports.postReview = (req, res) => {
+  const { userJwt, comment, placeId, rating } = req.body;
+  decodeJwt(userJwt)
```
Comment (AndriiDruzhytskyi) [inline controllers/reviews-controller.js]: We should decide which async functions style we will use: promise or async\await.

File: middlewares/decode-token.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const jwt = require('jsonwebtoken');
+const { jwtConf } = require('../config/config');
+
+module.exports = decodeJwt = async (token) => {
+  try {
```
Comment (AndriiDruzhytskyi) [inline middlewares/decode-token.js]: We already have Token service for any users tokens manipulations, please move this method to our Token service  and ipmort it where you need.

**Review by victorpoliushko — APPROVED** [review]

**Review by AndriiDruzhytskyi — COMMENTED** [review]
You should check if place with this id exist  before save new review to database. Also you should validate rating range too.
Screenshot from 2019-10-19 14-24-49
Screenshot from 2019-10-19 14-32-11

File: controllers/reviews-controller.js (hunk 1 above)
Comment (Dmitry528) [reply controllers/reviews-controller.js]: I rewrite this on async/await, and about rating, we have a 5 start for rating and user have choose between 0 or 5, and about check if place with this id exist before save new review to database, --- this is not my task, or no ?

File: middlewares/decode-token.js (hunk 2 above)
Comment (Dmitry528) [reply middlewares/decode-token.js]: Okay, today i will replace this method to token-service file

File: controllers/reviews-controller.js (hunk 1 above)
Comment (Dmitry528) [reply controllers/reviews-controller.js]: but about rating, okay i add validation

**Review by Viktor-Hordynskyi — APPROVED** [review]
Verified, works good, review sent

**Review by AndriiDruzhytskyi — APPROVED** [review]

**Pull request merged**